sys/mips/malta/malta_machdep.c

```diff
@@ -317,7 +355,16 @@ platform_start(__register_t a0, __register_t a1,  __register_t a2,
 			printf("%s ", (char*)(intptr_t)argv[i]);
 		printf("\n");
 	}
-
+	/*
+	 * parse the command line arguments passed e.g. by QEMUs -append option
+	 *
+	 * TODO: start parsing at index 1? QEMU passes -kernel option as
+	 * argv[0] so we currently call kern_setenv("/path/to/kernel", "1")
+	 * Or do other loaders/emulators pass arguments starting at index 0?
+	 */
```

Comment:
Cosmetic change request:

In general, per style(9), block comments should have one blank line before them, and the contents of the comments should be complete sentences with suitable capitalisation and punctuation. Could you adjust the first sentence here to begin with "Parse" and have a full stop at the end?